tests/integration/Observers/Post/StatusChangeTest.php

```diff
@@ -148,22 +149,45 @@ public function testCanUnpublishAndRepublishPostWithSameSlug()
         ]);
 
         $redirects = $this->findAllRedirects();
+        /* @var \Bonnier\WP\Redirect\Models\Redirect $lastRedirect */
+        $lastRedirect = $redirects->last();
+
         $this->assertCount(1, $redirects);
         $this->assertRedirect(
             $post->ID,
-            $redirects->last(),
+            $lastRedirect,
             '/dinosaur/carnivorous/t-rex',
             '/dinosaur/carnivorous',
             'post-draft'
         );
 
+        // Create a duplicate redirect but in another language to ensure it is not deleted on republish
+        $duplicatedRedirect = Redirect::createFromArray([
+            'from' => $lastRedirect->getFrom(),
+            'to' => $lastRedirect->getTo(),
+            'code' => $lastRedirect->getCode(),
+            'locale' => 'da',
```

Comment:
Default locale is da - use another locale to ensure, no unique key violations happens.
```suggestion
            'locale' => 'nb',
```